src/components/Sum.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum(props) {
```

Comment:
```suggestion
function Sum({ a, b }) {
```
Делай деструктуризацию props